iwyu.cc

```diff
@@ -3596,6 +3596,11 @@ class IwyuAstConsumer
     ParseFunctionTemplates(context.getTranslationUnitDecl());
 
     TraverseDecl(context.getTranslationUnitDecl());
+
+    // Check if any unrecoverable errors have occurred
```

Comment:
Comments should be sentences including full stop.

Reply:
Fixed in latest change.